recipe/meta.yaml

```diff
@@ -74,7 +74,6 @@ test:
         "avcodec",
         "avdevice",
         "swresample",
-        "postproc",
```

Comment:
just wondering why? postproc is enabled, does that mean the lib should be there?

Reply:
Because `postproc` will be part of `libavcodec.so`, it doesn't compile to its own library.

Reply:
thanks